### PR TITLE
fix(ChartCards): translate group and total tooltip labels

### LINKED ISSUE #1912: [ChartCards] tooltip Group and Total labels are not translated
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

tooltip Group and Total labels are not translatable through i18n

### To Reproduce

1. Go to TimeSeriesCard or BarChartCard and see that you can't set i18n on the tooltip

### Expected behavior

Should be translatable

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #1912

**Summary**

- Updating our i18n props to use this feature: https://github.com/carbon-design-system/carbon-charts/pull/826
- Adds i18n `tooltipGroupLabel` and `tooltipTotalLabel` to BarChartCard
- Adds i18n `tooltipGroupLabel` to TimeSeriesCard (doesn't use Total)

**Acceptance Test (how to verify the PR)**

- Start server locally and add the above i18n props to each card and see that the string is being translated correctly. Note: 'Total' is only shown in stacked bar charts when you hover over the total bar axis

### CODE REVIEW

**Comment by sstone2423** [comment]
I'm not sure why, but I'm getting this console error which is causing PieChartCard's color tests to fail

```
    console.error
      Missing CSS styles for Carbon Charts. Please read the Carbon Charts getting started guide.

      at node_modules/@carbon/charts/bundle.js:15:146057
      at Timeout.task [as _onTimeout] (node_modules/jsdom/lib/jsdom/browser/Window.js:391:19)
```

**Review by scottdickerson — APPROVED** [review]
LGTM

File: yarn.lock [hunk 1]
```diff
@@ -11481,9 +11481,8 @@ inherits@2.0.3:
   resolved "https://registry.yarnpkg.com/inherits/-/inherits-2.0.3.tgz#633c2c83e3da42a502f52466022480f4208261de"
 
 ini@^1.3.4, ini@^1.3.5, ini@~1.3.0:
-  version "1.3.7"
-  resolved "https://registry.yarnpkg.com/ini/-/ini-1.3.7.tgz#a09363e1911972ea16d7a8851005d84cf09a9a84"
-  integrity sha512-iKpRpXP+CrP2jyrxvg1kMUpXDyRUFDWurxbnVT1vQPx+Wz9uCYsMIqYuSBLV+PAaZG/d7kRLKRFc9oDMsH+mFQ==
+  version "1.3.5"
+  resolved "https://registry.yarnpkg.com/ini/-/ini-1.3.5.tgz#eee25f56db1c9ec6085e0c22778083f596abf927"
```
Comment (tay1orjones) [inline yarn.lock:11485]: Any idea where this came from? I think we should revert it - 1.3.5 contains a security vulnerability and was updated to 1.3.7 via dependabot in https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1878

**Review by tay1orjones — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (sstone2423) [reply yarn.lock:11485]: I think thats a mistake. I originally updated carbon charts and then reverted and it looks like that slipped in. I'll swap it back

Comment (sstone2423) [reply yarn.lock]: Fixed!

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**